frappe/core/doctype/translation/test_translation.py

```diff
@@ -110,3 +111,8 @@ def create_translation(key, val):
 	translation.translated_text = val[1]
 	translation.save()
 	return translation
+
+
+def clear_translation_cache():
+	frappe.local.lang_full_dict = None
+	frappe.cache().delete_key("lang_full_dict", shared=True)
```

Comment:
An alternative to this is doing something like following, for tests:
https://github.com/frappe/frappe/pull/17304/files#diff-e4c5479669e595d33f505872bedf9d52b2570e2a75b4e8f8cccc3834d3cd86a2R278-R279